psp/scripts/eval_model.py

```diff
@@ -1,114 +1,113 @@
-import argparse
 import datetime
-import pathlib
+import importlib
+import pickle
 
-# import altair as alt
+import click
 import numpy as np
 import pandas as pd
 import tqdm
 
-from psp.data.data_sources.pv import PvDataSource
-from psp.ml.metrics import MeanRelativeError, Metric, mean_absolute_error
-from psp.ml.models.previous_day import PreviousDayPvSiteModel
+from psp.ml.dataset import split_train_test
+from psp.ml.metrics import Metric, mean_absolute_error
+from psp.ml.models.base import PvSiteModel
 from psp.ml.training import make_data_loader
+from psp.scripts._options import (
+    exp_config_opt,
+    exp_name_opt,
+    exp_root_opt,
+    num_workers_opt,
+)
+from psp.utils.interupting import continue_on_interupt
 
 METRICS: dict[str, Metric] = {
-    "mre_cap=1": MeanRelativeError(cap=1),
+    # "mre_cap=1": MeanRelativeError(cap=1),
     "mae": mean_absolute_error,
 }
 
 
-def parse_args():
-    parser = argparse.ArgumentParser(__doc__)
-    parser.add_argument("-p", "--pv-data", required=True, help=".netcdf PV data")
-    parser.add_argument(
-        "-n",
-        "--exp-name",
-        help="name of the experiment (used as the name of the dictionary)",
+@click.command()
+@exp_root_opt
```

Comment:
is it worth expanding these, witha. description and type,

Reply:
They are defined in `_options.py` with `type` and `help`